2018/02.md

```diff
@@ -0,0 +1,326 @@
+autoscale: true
+theme: Huerta, 1
+
+# [fit] The Latest in PHP
+## [fit] FEB EDITION
+
+---
+
+# [fit] Presented by
+## [fit] CHRIS DEATON
+## [fit] @flashadvocate
+
+---
+
+# PHP Releases
+
+VERSIONS
+
+v7.2.2 - Bug fixes
+v7.1.14 - Bug fixes
+v5.6.33 - Bug fixes
+
+Links:
+
+* http://php.net/ChangeLog-7.php
+* http://php.net/ChangeLog-5.php
+
+---
+
+# PHP RFCs
+
+* Pending Implementation
+ * Flexible Heredoc and Nowdoc syntaxes
+ * Shorthand null coalesce equal operator (no more redundancy)
+
+ 	```php 
+	 	// bad
+	 	$this->thing['inside']['more'] = $this->thing['inside']['more'] ?? 'falsy value'`
+	 	// good
+	 	$this->thing['inside']['more'] ??= 'falsy value'
+ 	```
+ 
+ * `E_DEPRECATED` inconsistent cast names 
+
+ 		```php
+ 		(integer) // use (int)
+ 		(boolean) // use (bool)
+ 		(double) // use (float)
+ 		(real) // use (float)
+ 		``` 
+
+Links
+* [https://wiki.php.net/rfc](https://wiki.php.net/rfc)
+* [https://twitter.com/phprfcbot](https://twitter.com/phprfcbot)
+* [https://php-rfc-watch.beberlei.de/](https://php-rfc-watch.beberlei.de/)
+
+---
+
+# PHPUnit
+
+* PHPUnit 6.5 (Dec 2017) - [Release Notes](https://github.com/sebastianbergmann/phpunit/wiki/Release-Announcement-for-PHPUnit-6.5.0)
+
+Links: 
+
+* https://github.com/sebastianbergmann/phpunit/wiki
+
+---
+
+# Framework: Laravel
+
+* Laravel 5.6[.3]
+ * Improved logging "stacks" via `config/logging.php`
+ * Dynamic rate limiting - ability to set user-specific rate limits
+ * Broadcast channel classes
+ * API Controller Scaffolding (omits view-specific methods like `create`, `edit`)
+ * Aliases for Blade components! - used like directives once defined
+
+Links: 
+
+* https://github.com/laravel/framework/releases
+* https://laravel-news.com/category/news
+
+---
+
+# Framework: Symfony
+
+- The end of Symfony Standard - [Article](https://symfony.com/blog/the-end-of-the-symfony-standard-edition) 
+	- New project by Fabian - [Flex](https://symfony.com/doc/current/setup.html)
```

Comment:
I didn't see any useful information about Flex on that page. Perhaps https://symfony.com/doc/current/setup/flex.html ?

Reply:
Good catch! Thanks Jonathan